### PR TITLE
feat(ui5-avatar-group): Implement accessibility specification

### PR DESCRIPTION
**Pull request by dobrinyonkov**
Fixes: #2745

* new property `ariaHaspopup` added to `ui5-avatar` and `ui5-avatar-group`
* aria-haspopup attribute of the overflow button or the avatars container renders (or not) depending on the value of
the `ariaHaspopup` property and the `type` property
* interactive `ui5-avatar` now has role `button`

### CODE REVIEW

File: packages/main/src/Avatar.js
Comment (fifoosid) [inline packages/main/src/Avatar.js:213]: </ul>

File: packages/main/src/Avatar.js
Comment (fifoosid) [inline packages/main/src/Avatar.js:206]: Add </code> to the end of every list item

File: packages/main/src/AvatarGroup.js
Comment (fifoosid) [inline packages/main/src/AvatarGroup.js:109]: </ul>

File: packages/main/src/AvatarGroup.js
Comment (fifoosid) [inline packages/main/src/AvatarGroup.js:121]: </ul>

File: packages/main/src/AvatarGroup.js
Comment (fifoosid) [inline packages/main/src/AvatarGroup.js:241]: When focused, the ...

**Review by fifoosid — CHANGES_REQUESTED** [review]
From code POV, looks good to me. I have left some small comments regarding the docs. You can fix the build as well, because there are some lint errors:
image

Once you fix the comments and get a review from @SAP/ui5-webcomponents-topic-p we can merge it.

**Review by fifoosid — COMMENTED** [review]
There is a merge conflict, which needs to be resolved

**Review by fifoosid — COMMENTED** [review]
Looks good to me, someone from @SAP/ui5-webcomponents-topic-p should approve and we can merge

**Review by ilhan007 — DISMISSED** [review]

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**
